content/issues/2023-06-14-ci-maintenance.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Maintenance on ci.jenkins.io
+date: 2023-06-14T11:00:00-00:00
```

Comment:
```suggestion
date: 2023-06-14T12:00:00-00:00
```
(According to the jenkins security event?)

Reply:
In my mind it was 13h00 like previous one. Good catch.